README.md

```diff
@@ -1,30 +1,57 @@
+7. Have some short meaningfull output in the UI and detailed logs in a file. There should be a dedicated 'View logs' button to open log folder for convinience.
+8. Introduce profiles that will contain list of repositories that should be checked for synchronization.
+9. Build application
+10. Run application
 
 ### Implementation
 
-This will be a desktop application developed using WPF. The configuration file mentioned earlier will be local in the first iteration, in the future it is planned to load it from a remote repository. 
-It is planned to use the git application to work with repositories because the pool of scheduled tasks does not require complex commands. List of commands that are planned to be used are : branch, checkout, pull, stash. 
+This will be a desktop application developed using WPF targeting .NET 6. It is planned to use the Git CMD application to work with repositories because the pool of scheduled tasks does not require complex commands. List of commands that are planned to be used are : branch, checkout, pull, stash.
+
+The configuration file mentioned earlier will be local in the first iteration, in the future it is planned to load it from a remote repository. 
+
 All information will be presented in a table. One line in it corresponds to one project. 
-To select the necessary branch there will be a ComboBox with an available selection. After branch selection, SourceExplorer will check if the branch is presented locally. If not - it will perform a pull operation. 
-Also, there will be a ComboBox for build mode.
-For each operation, buttons are provided to perform it. If action is required to be performed on multiple projects, user must click on corresponding CheckBoxes and then on an action button.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/fulltable.png)
+
+To select the necessary branch there will be a ComboBox with an available selection. 
+After branch selection, SourceExplorer will check if the branch is presented locally. 
+If not - it will perform a pull operation.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/branchselection.png)
+Also, there will be a ComboBox for build mode. <br />
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/buildmode.png)  <br />
+For each operation, buttons are provided to perform it. If action is required to be performed on multiple projects, user must click on corresponding CheckBoxes and then on an action button in header of the table.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/choosing.PNG)
+
 ### Information security
 
-Authorization in the version control system will be implemented via open auth.
+Authorization in the version control system will be implemented via open auth. The user will be logged into the git on the local device.
 
-### Validation
+### Implementation iterations
 
+Tasks of the first iteration:
```

Comment:
Please describe also the second iteration.

Reply:
added some information